exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/flatten/FlattenRecordBatch.java

```diff
@@ -99,6 +100,22 @@ private void clear() {
     }
   }
 
+  public enum Metric implements MetricDef {
+    NUM_INCOMING_BATCHES,
+    AVG_INPUT_BATCH_SIZE,
```

Comment:
Not clear what units `SIZE` is in. Maybe `AVG_INPUT_BATCH_BYTES` (if it is bytes)?